docs/management/admin/endpoint-event-capture.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[endpoint-event-capture]]
+[chapter]
+= Event capture and {elastic-defend}
+
+{elastic-defend} collects selective data on system activities in order to detect and prevent as many threats as possible, while balancing storage and performance overhead. To that end, {elastic-defend} isn't designed to provide a complete capture of all system events. The event data that {elastic-defend} generates might be aggregated, truncated, or deduplicated as needed to optimize threat detection and prevention.
+
+You can supplement {elastic-defend}'s protection capabilities with additional {integrations-docs}[Elastic integrations] and tools that provide more visibility and historical data. Consult the following sections to expand data collection for specific system events.
+
+
+[discrete]
+== Network port creation and deletion
+
+{elastic-defend} tracks TCP connections. If a port is created but no traffic flows, no events will be generated.
+
+For complete capture of network port creation and deletion, consider capturing Windows event ID 5158 using the {integrations-docs}/winlog[Custom Windows Event Logs] integration.
+
+[discrete]
+== Network in/out connections
+
+{elastic-defend} tracks TCP connections, which don't include network in/out connections.
```

Comment:
Optional edit for clarity.

```suggestion
{elastic-defend} tracks TCP connections, which don't include network in or out connections.
```

Reply:
Seems like "in/out" is a sorta common IT term so probably keep this as is.